styles.css

```diff
@@ -0,0 +1,373 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: #FFF;
+    margin: 15px;
+}
+
+h1, h2, h3, h4 {
+    font-family: 'Roboto Slab', serif;
+    color: #353535;
+    margin: 10px 0 10px 10px;
+}
+
+
+
+p, li {
+    font-family: 'Abel', sans-serif;
+    padding: 20px 15px;
+    font-size: 1.2rem;
+    color: #575A61;
+}
+
+img {
+    max-width: 100%;
+    margin: 0;
+}
+
+a {
+    text-decoration: none;
+    color: #353535;
+    font-weight: bolder;
+}
+
+a:visited {
+    color: #353535;
+}
+
+
+/* **** Navigation ***** */
+
+.header-name {
+    font-family: 'Roboto Slab', serif;
+    font-weight: 500;
+}
+
+.navbar {
+    display:flex;
+    flex-direction: column;
+}
+
+.navbar ul {
+    list-style-type: none;
+}
+
+.navbar a {
+    text-decoration: none;
+    color: #FFFFFF;
+    font-weight: 200;
+}
+
+.nav-link {
+    font-family: 'Abel', sans-serif;
+    padding: 5px 10px;
+    /* border: 0.5px #353535 solid; */
+    text-decoration: none;
+    background-color: #6999F3;
+    text-align: center;
+    margin: 4px 8px;
+    box-shadow: 3px 3px 5px #e6ecf7;
+}
+
+/* **** About **** */
+
+.about-liz {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+}
+
+.about-text3 {
+    width: 100%;
+}
+
+.main-pic, .about2-pic, .about3-pic {
```

Comment:
This is good, adding the same CSS to three classes. Another way to do this would be have one class for this CSS and then put multiple classes on each of these pics.